sarathi/sarathi.py

```diff
@@ -86,6 +86,13 @@ async def on_error(event, *args, **kwargs):
             raise Exception(
                 "Error encountered: {} x {} x {}".format(event, args, kwargs))
 
+@bot.command(name="fortune")
+async def fortune(ctx,*args):
+
+    arg_list = list(args)
+    fortunes = til.print_fortune(arg_list)
```

Comment:
When naming functions, think about *what they do* instead of what you use their output for.

In this case, the function `print_fortune` doesn't *print* the fortune, it instead returns the `fortune` command's output to the function's caller. So a more appropriate name, among others, would be `call_fortune` or `run_fortune`. You can choose other names, but ensure that they convey the purpose of the function.